src/helpers/getDomain.js

```diff
@@ -8,7 +8,7 @@ import { isProduction } from './isProduction';
  */
 export const getDomain = () => {
   const prodUrl = 'http://sopra-fs21-group-12-server.herokuapp.com'; // TODO: insert your groups heroku prod url for server (once deployed)
-  const devUrl =  'http://sopra-fs21-group-12-server.herokuapp.com';
+  const devUrl =  'http://localhost:8080';
```

Comment:
wemmer das in master denne ha?

Reply:
jaa isch mir egal, ig bis eigentlich immer wieder ufe Heroku server am ändere damit ig mit Bilder cha schaffe 😅

Reply:
ja ebbe dorom meini, denn sell nomol pusht wärde aber metm heroku link 👍

Reply:
jo und ich bruche immer de localhost. vo dem her chonds würklich ned druf ah. pushe wege dem jetzt ned extra nomal;)